newIDE/app/src/MainFrame/EditorContainers/ExternalLayoutEditorContainer.js

```diff
@@ -110,6 +111,34 @@ export class ExternalLayoutEditorContainer extends React.Component<
     }
   }
 
+  onEventsBasedObjectChildrenEdited() {
+    const { editor } = this;
+    if (editor) {
+      // Update every custom objects because some custom objects may include
+      // the one actually edited.
+      editor.forceUpdateCustomObjectRenderedInstances();
+    }
+  }
+
+  onSceneObjectEdited(scene: gdLayout, objectWithContext: ObjectWithContext) {
+    const { editor } = this;
+    const externalLayout = this.getExternalLayout();
+    if (!externalLayout) {
+      return;
+    }
+    if (
+      externalLayout.getAssociatedLayout() !== scene.getName() &&
+      !objectWithContext.global
+    ) {
+      return;
+    }
+    if (editor) {
+      // Update every custom objects because some custom objects may include
+      // the one actually edited.
```

Comment:
Same as the other:

"Update instances of the object, as it was modified in another editor."